### PR TITLE
[nearlib] add api for schedule function call + integration test

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: nearlib/test/integration.test.js [hunk 1]
```diff
@@ -69,21 +70,54 @@ test('deploy contract and make function calls', async () => {
...
+    }
+    const scheduleResult = await nearjs.scheduleFunctionCall(
+        0,
+        aliceAccountName,
+        "test_contract",
+        "near_func_setValue", // this is the function defined in hello.wasm file that we are calling
+        setArgs);
+    waitForNonceToIncrease(accountBeforeScheduleCall);
+
+    for (let i = 0; i < TEST_MAX_RETRIES; i++) {
```
Comment (vgrichina) [inline nearlib/test/integration.test.js]: Extract for / try / catch pattern with retries into common helper used by different tests?

File: nearlib/near.js [hunk 2]
```diff
@@ -28,6 +28,25 @@ class Near {
...
+    /**
+     * Schedules an asynchronous function call.
+     */
+    async scheduleFunctionCall(amount, sender, contractAccountId, methodName, args) {
+        if (!args) {
+            args = {};
+        }
+        const serializedArgs =  Array.from(BSON.serialize(args));
+        const response = await this.nearClient.submitTransaction('schedule_function_call', {
+            // TODO(#5): Need to make sure that big ints are supported later
```
Comment (vgrichina) [inline nearlib/near.js]: given the state of browser support for bigints (even Chrome doesn't support them in JSON yet), it's best to send them as strings basically.

BTW, not sure whether you really wanted to link this issue: https://github.com/nearprotocol/nearcore/issues/5

Comment (janedegtiareva) [reply nearlib/near.js]: Thanks, this was a copy paste from contract helper which lost relevance being moved to nearcore. Removed comment.

File: nearlib/test/integration.test.js (hunk 1 above)
Comment (janedegtiareva) [reply nearlib/test/integration.test.js]: I'd like to get rid of retries eventually, but I suppose that we can have this helper for now.

**Pull request merged**